recipes/pocketsphinx/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pocketsphinx-python" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/bambocher/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: d56244340161847cd978fdf9aba0d477bf5f99fcb3019f5b10dea921d078e23a 
+  - git_url: https://github.com/cmusphinx/pocketsphinx
+    git_tag: 1fdc9ccb637836d45d40956e745477a2bd3b470a
+    folder: deps/pocketsphinx
+  - git_url: https://github.com/cmusphinx/sphinxbase
+    git_tag: 01392d23f8b45f4e0567c46057cf1149c9a124e7
```

Comment:
Also, it doesn't seem like this has had a release....

Reply:
can still use a github archive

Reply:
Yeah, definitely, just not a git url and tag